db-browser-for-sqlite: Bump version from 3.13.0 to 3.13.1

lucydodo opened the pull request:
Related: https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.13.1

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

Any comments or review are welcome. Thank you.

khipp commented:
What's the rationale behind dropping the legacy version as opposed to just bumping the `version` in the block for macOS Big Sur and newer?

lucydodo commented:
Hi @khipp,
Sorry to forgot to mention something important.
Our v3.13.0 version had a bug that prevented the app from running on Catalina and earlier versions, so I temporarily changed it to download an older version on macOS Catalina and earlier versions: #181272
However, with this new release, I've removed that block because we've fixed the issue.

khipp reviewed (approved):
Thank you!
